### PR TITLE
Rename communities to collections

### PR DESCRIPTION
**Pull request by ash3rz**
Just a blanket rename.  I did update the i18n strings to reference one string `$t(featureName)` or `$t(featureName_plural)` so if we change our minds about the name for this feature, we can change it there in the future.

### CODE REVIEW

File: stories/collections/CollectionMocks.js [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-            description: "App Community",
+            description: "App Collection",
             name: "Last Wednesday Society",
             type: "group",
             member: false,
             extension: "Last Wednesday Society",
             id: "f5e01bb0ba26449c8a25032d95e92c29",
             display_extension: "Last Wednesday Society",
-            display_name: "iplant:de:qa:communities:Last Wednesday Society",
+            display_name: "iplant:de:qa:collections:Last Wednesday Society",
```
[the remark this answers is not in the document]
Comment (ash3rz) [reply stories/collections/CollectionMocks.js:51]: Good catch!  I think it makes sense to have the mocks match what the service will provide, so I'll revert this back.

**Comment by sri2k1us** [comment]
LGTM. We will also need to update configs related to the new `de-mailer` (emailservice.yml) service that references DE pages.

**Review by sri2k1us — APPROVED** [review]
:shipit:

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**